apps/freelance/src/pages/SignupPage/styles.ts

```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
+
+export const Wrapper = styled.div`
+margin-top: 50px;
+margin-left: 100px;
+`;
```

Comment:
something with eslint
```
export const Wrapper = styled.div`
    margin-top: 50px;
    margin-left: 100px;
`;
```
it should look like it

Reply:
checked